projects/kit/directives/lazy-loading/lazy-loading.directive.ts

```diff
@@ -26,14 +26,17 @@ export class TuiImgLazyLoading {
     protected animation = signal('tuiSkeletonVibe ease-in-out 1s infinite alternate');
     protected background = signal('var(--tui-background-neutral-2)');
 
-    protected readonly $ = this.loading$
-        .pipe(takeUntilDestroyed())
-        .subscribe((src) => this.src.set(src));
+    protected readonly $ = this.supported
```

Comment:
!this.supported && this.loading.pipe(...)

Reply:
fixed